src/views/Diff.vue

```diff
@@ -0,0 +1,180 @@
+<template>
+	<div class="bg-gray-300">
+		<ResultHeader
+			title="Podobnosť dokumentov:"
+			:documents="[documents.textA.name, documents.textB.name]"
+		/>
+		<div class="md:flex py-4 md:py-4">
+			<div
+				class="md:flex1 rounded-md md:rounded-lg mx-4 md:mt-0 md:mr-2 md:ml-4 md:w-1/2 lg:ml-12"
+			>
+				<div
+					class="flex items-center justify-between px-2 py-1 md:p-2 md:h-12 bg-primary-500 text-white text-lg rounded-t-md md:rounded-t-lg"
+				>
+					<p class="mx-auto">
+						{{ documents.textA.name }}
+					</p>
+				</div>
+				<div
+					v-html="doneA"
+					class="px-4 py-1 pt-1 md:pt-2 md:p-4 bg-white text-justify rounded-b-md md:rounded-b-lg shadow-md"
+				/>
+			</div>
+			<div
+				class="md:flex1 rounded-md md:rounded-md mt-4 mx-4 md:mt-0 d:mr-4 md:ml-2 md:w-1/2 lg:mr-12"
+			>
+				<div
+					class="flex items-center justify-between px-2 py-1 md:p-2 md:h-12 bg-primary-500 text-white text-lg rounded-t-md md:rounded-t-lg"
+				>
+					<p class="mx-auto">
+						{{ documents.textB.name }}
+					</p>
+				</div>
+				<div
+					v-html="doneB"
+					class="px-4 py-1 pt-1 md:pt-2 md:p-4 bg-white text-justify rounded-b-md md:rounded-b-lg shadow-md"
+				/>
+			</div>
+		</div>
+	</div>
+</template>
+
+<script>
+import ResultHeader from '../components/Result/ResultHeader';
+import { escape } from 'lodash';
+import retry from '@/functions/retry.function';
+
+export default {
+	components: {
+		ResultHeader,
+	},
+	data: function () {
+		return {
+			id: undefined,
+			doneA: '',
+			doneB: '',
+			documents: {},
+		};
+	},
+	mounted: function () {
+		// initial spinner
+		this.$store.dispatch('setLoading', true);
+
+		this.id = this.$route.params.diff;
+
+		retry(
+			() => this.fetchResult(this.id),
+			(result) => result.length !== 0
```

Comment:
Please avoid having negative conditions. Positive conditions are much easier to read. 
Example: `Length of result IS NOT zero` vs `More than zero documents (Length of result is greater than 0)`